### PR TITLE
Store Pictures in MySql Blob Field

### PR DESCRIPTION
**Pull request by b6025**

### CODE REVIEW

File: util/PhotoMigration.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+    private String _patPhotoPath;
+    private String _encPhotoPath;
+    private String _dbUrl;
+    private String _dbUser;
+    private String _dbPass;
+
+    public PhotoMigration(String dbUrl, String dbUserName, String dbPassword, String patientPhotoPath, String encounterPhotoPath)
+    {
+        _dbUrl = dbUrl;
```
Comment (kevinzurek) [inline util/PhotoMigration.java]: @b6025 what should i use for dbUrl? I tried:

localhost
and
jdbc:mysql://localhost/test?characterEncoding=UTF-8

Both say "No suitable driver found"

**Comment by kevinzurek** [comment]
I made 1 change in the application code so empty folders don't get created if saving to the database is turned on. I think it should be okay:
https://github.com/kevinzurek/femr/commit/7978035fb6c905bf8311a5ed87241551f5b7ac87

Other than that i'm just trying to give the photo upload utility the proper parameters. Thank you for your help getting this done!!!!

**Comment by b6025** [comment]
No worries.  I think that database error is because it can't find the mysql
jdbc driver.  When you run the app via the commandline, you'll need to
specify the classpath (java -classpath [path to mysql jdbc], or set the
CLASSPATH environment variable) that contains the driver (it's loaded at
runtime).  Here's an article I found that may be helpful:
http://stackoverflow.com/questions/8146793/no-suitable-driver-found-for-jdbcmysql-localhost3306-mysql

-Brandon

**Comment by kevinzurek** [comment]
@b6025 Error switched up a bit after setting the classpath:

[18:06 kevin@kevinPad: ~/Git/femr/util (pr591-7978035*)]$ java PhotoMigration "jdbc:mysql://localhost:3306/test" usr pw ~/Git/femr/Upload/Pictures/Patients ~/Git/femr/Upload/Pictures/PatientEncounters/
Error: Could not find or load main class PhotoMigration
[18:06 kevin@kevinPad: ~/Git/femr/util (pr591-7978035*)]$ echo $CLASSPATH
/home/kevin/Git/femr/util/mysql-connector-java-5.1.41-bin.jar

Working on figuring this out right now

**Comment by kevinzurek** [comment]
Successfully converted pictures and merged manually into master:

ef241e061502e83a650a76b99fa78179729f3d20

Thank you @b6025 !!

https://teamfemr.atlassian.net/browse/FEMR-291

**Pull request closed without merging**